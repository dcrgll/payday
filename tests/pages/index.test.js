import { render, screen } from '@testing-library/react'
import Home from '@/pages/index'

it('renders a heading', () => {
  const { container } = render(<Home />)

  const heading = screen.getByRole('heading', {
    name: 'hello'
  })

  expect(heading).toBeInTheDocument()

  expect(container).toMatchSnapshot()
})
