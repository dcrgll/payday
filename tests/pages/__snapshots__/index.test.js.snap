// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a heading 1`] = `
<div>
  <h1>
    hello
  </h1>
</div>
`;
